### PR TITLE
fix: Do not add history entry on host details page

### PR DESCRIPTION
**Pull request by gkarat**
(no jira)

This improves how the system details page change the browser history on navigating between tabs. Users used to stay on the same host details page if they clicked on different tabs like Advisor or Patch and then tried to click on browser's Back/Forward button. This is not really comfortable, and pressing Back should navigate to the previous page instead. To do this, the current browser's history entry should be replaced instead of adding a new one.

## How to test

Go to /inventory and navigate to any host details page. Click on tabs, then click Back in your browser and make sure you are navigated back to /inventory table (users used to stay on host details page which is not expected behaviour).

### CODE REVIEW

**Review by Fewwy — APPROVED** [review]

File: src/routes/InventoryDetail.js [hunk 1]
```diff
@@ -113,9 +112,12 @@ const Inventory = () => {
   useEffect(() => {
     if (searchParams.get('appName') === null) {
       searchParams.set('appName', appList[0].name);
-      navigate({
-        search: searchParams.toString(),
-      });
+      navigate(
```
Comment (bastilian) [inline src/routes/InventoryDetail.js:115]: You can use `useSearchParams` for this.

Comment (gkarat) [reply src/routes/InventoryDetail.js:119]: Good suggestion:) added to the PR

**Comment by gkarat** [comment]
Again the __coverage__ false negative in cy tests... The rest is fine so assuming tests pass

**Pull request merged**